PROJECT

```diff
@@ -0,0 +1,23 @@
+domain: openebs.io
```

Comment:
What is this file used for? Is it mandatory to include in the project?

Reply:
This is used by Kubebuilder and Operator-SDK. They expect this to be there